node_modules/oae-library/lib/api.search.js

```diff
@@ -81,7 +83,22 @@ module.exports.registerLibrarySearch = function(searchName, resourceTypes, optio
                     return callback({'code': 401, 'msg': 'You are not authorized to access this library search feed'});
                 }
 
-                var query = SearchUtil.createQueryStringQuery(opts.q);
+                var query = {
+                    'bool': {
+                        'should': [
+                            // Search on basic metadata properties such as display name or description
+                            SearchUtil.createQueryStringQuery(opts.q),
+
+                            // Search on content comments or discussion messages
+                            SearchUtil.createHasChildQuery(ContentConstants.search.MAPPING_CONTENT_COMMENT, SearchUtil.createQueryStringQuery(opts.q, ['body']), 'max'),
```

Comment:
Does this mean that both content and discussion library search queries would have redundant clauses? Is that something we want to avoid?